diff --git a/sns/src/main/java/board/controller/BoardController.java b/sns/src/main/java/board/controller/BoardController.java
index d84f331..c1af100 100644
--- a/sns/src/main/java/board/controller/BoardController.java
+++ b/sns/src/main/java/board/controller/BoardController.java
@@ -4,6 +4,7 @@ import java.io.File;
 import java.io.IOException;import java.io.Writer;
 
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 import javax.validation.Valid;
 
@@ -24,6 +25,7 @@ import org.springframework.web.multipart.MultipartFile;
 import board.domain.BoardVO;
 import board.service.BoardService;
 import member.domain.MemberVO;
+import member.service.MemberService;
 
 @Controller
 @SessionAttributes("boardVO")
@@ -47,25 +49,36 @@ public class BoardController {
 	}
 	//새 글 작성을 위한 요청을 처리
 	@RequestMapping(value="/board/write", method=RequestMethod.GET)
-	public String write(Model model,HttpServletRequest req) {
-		req.getAttribute("member");
+	public String write(Model model,HttpServletRequest req ,HttpServletResponse resp) {
+		HttpSession session = req.getSession();
+		session.getAttribute("member");
+		MemberVO memberVO = (MemberVO) session.getAttribute("member");
 		model.addAttribute("boardVO", new BoardVO());
 		return "/board/write";
 		
 	}
 	//새 글 등록을 위한 요청을 처리
 	@RequestMapping(value="/board/write", method=RequestMethod.POST)
-	public String write(@Valid BoardVO boardVO, HttpServletRequest req,BindingResult bindingResult, @RequestParam("filename")MultipartFile uploadfile)throws IOException {
+	public String write(@Valid Model model,BoardVO boardVO, HttpServletRequest req,BindingResult bindingResult, @RequestParam("filename")MultipartFile uploadfile)throws IOException {
 			if(bindingResult.hasErrors()) {
 				return "/board/write";
 			}
+			model.addAttribute("boardVO", new BoardVO());
+			HttpSession session = req.getSession();
+			session.getAttribute("member");
+			MemberVO memberVO = (MemberVO) session.getAttribute("member");
 			
 			System.out.println();
 			boardVO.setImages(uploadfile.getOriginalFilename());
-			req.getAttribute("member");
 			String path = "C:\\Users\\Yeonheung\\springwork\\sns\\src\\main\\webapp\\images";
 			uploadfile.transferTo(new File(path, uploadfile.getOriginalFilename()));
-			
+			System.out.println(boardVO.getContent());
+			System.out.println(boardVO.getNum());
+			System.out.println(boardVO.getTitle());
+			System.out.println(memberVO.getWriter());
+			System.out.println(memberVO.getPass());
+			memberVO.getWriter();
+			memberVO.getPass();
 			boardService.write(boardVO);
 			return "redirect:/board/list";
 	}
diff --git a/sns/src/main/resources/config/sqlmap-board.xml b/sns/src/main/resources/config/sqlmap-board.xml
index 6d63644..07415b8 100644
--- a/sns/src/main/resources/config/sqlmap-board.xml
+++ b/sns/src/main/resources/config/sqlmap-board.xml
@@ -22,11 +22,11 @@
 	</select> 
 	<insert id="insert" parameterType="boardVO">
 			INSERT INTO SNSBOARD
-			(num, pass, title, content, writer, regDate, images)
+			(num, title, writer, pass, content, regDate, images)
 			VALUES
 			(SNSBOARD_num.nextval,
-					#{pass}, #{title}, #{content}, #{writer}, SYSDATE, #{images})
-			<selectKey keyProperty="seq" resultType="Integer">
+					#{title}, #{writer}, #{pass}, #{content}, SYSDATE, #{images})
+			<selectKey keyProperty="num" resultType="Integer">
 					SELECT NVL(MAX(num), 0) FROM SNSBOARD
 			</selectKey>
 	</insert>
diff --git a/sns/src/main/resources/config/sqlmap-member.xml b/sns/src/main/resources/config/sqlmap-member.xml
index ad62bbc..b0aba16 100644
--- a/sns/src/main/resources/config/sqlmap-member.xml
+++ b/sns/src/main/resources/config/sqlmap-member.xml
@@ -25,7 +25,6 @@
 		SELECT * FROM SNSMEMBER WHERE WRITER
 	</select>
 
-
 	
 	<delete id="memberdeleteAll">
 			DELETE FROM SNSBOARD
diff --git a/sns/src/main/resources/spring-controller.xml b/sns/src/main/resources/spring-controller.xml
index 36430f9..ebf97e6 100644
--- a/sns/src/main/resources/spring-controller.xml
+++ b/sns/src/main/resources/spring-controller.xml
@@ -12,5 +12,4 @@
 	<bean id="memberController" class="member.controller.MemberController">
 			<property name="memberService" ref="memberServiceImpl"/>	
 	</bean>
-
 </beans>
diff --git a/sns/src/main/webapp/WEB-INF/view/board/list.jsp b/sns/src/main/webapp/WEB-INF/view/board/list.jsp
index fc662c3..6a850ab 100644
--- a/sns/src/main/webapp/WEB-INF/view/board/list.jsp
+++ b/sns/src/main/webapp/WEB-INF/view/board/list.jsp
@@ -147,7 +147,6 @@ butt{
    <div class="w3-panel w3-border-top w3-border-bottom">
     <a style= "text-align:left">${board.writer}</a><a style = "float:right">${board.regDate}</a>
     </div>
-     
       <div><img src="../images/${board.images}" width="auto" height="150" alt="no img"></div>
       <div>
       <h4>${board.content}</h4><!-- content,regdate 안됨 -->
@@ -165,11 +164,7 @@ butt{
        <a href="<c:url value="/board/edit/${board.num}"/>"><i class="fas fa-file" style ="font-size:25px"></i></a>
       &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
       
-      
-      
-      
-      
-      
+     
       <!-- 삭제팝업창으로 넘김버튼 -->
         <a href="<c:url value="/board/delete/${board.num}"/>"><i class="glyphicon glyphicon-remove" style ="font-size:25px"></i></a>
         </div>
diff --git a/sns/src/main/webapp/WEB-INF/view/board/read.jsp b/sns/src/main/webapp/WEB-INF/view/board/read.jsp
index 7b4beea..ce083e4 100644
--- a/sns/src/main/webapp/WEB-INF/view/board/read.jsp
+++ b/sns/src/main/webapp/WEB-INF/view/board/read.jsp
@@ -8,6 +8,7 @@
 <meta charset="UTF-8">
 <title>${boardVO.title}&nbsp;내용</title>
 </head>
+<!-- 검색 후 유저 목록 친구목록 구현 -->
 <body>
 		<table border="1">
 		
diff --git a/sns/src/main/webapp/WEB-INF/view/board/write.jsp b/sns/src/main/webapp/WEB-INF/view/board/write.jsp
index 0d06600..b8fe1fd 100644
--- a/sns/src/main/webapp/WEB-INF/view/board/write.jsp
+++ b/sns/src/main/webapp/WEB-INF/view/board/write.jsp
@@ -40,7 +40,6 @@ butt{
   <h1>제목 미정</h1>
   <p>부제 미정</p> 
 </div>
-
 <nav class="navbar navbar-expand-sm bg-dark navbar-dark">
   <a class="topnav-icons fa fa-home w3-left w3-bar-item w3-button" title="Home" style ="font-size:25px; color:white" href="<c:url value="/board/list"/>"></a>
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
@@ -70,7 +69,6 @@ butt{
        <c:if test="${not empty member.writer}">
       <a href="<c:url value='/member/logout'/>" class="w3-bar-item w3-button">로그아웃</a>
       <a href="#" class="w3-bar-item w3-button">내정보</a>
-      
       </c:if>
       </div>
       </div>
diff --git a/sns/src/main/webapp/images/Thumbs.db b/sns/src/main/webapp/images/Thumbs.db
index 6bc271a..0f1c5c7 100644
Binary files a/sns/src/main/webapp/images/Thumbs.db and b/sns/src/main/webapp/images/Thumbs.db differ
diff --git a/sns/src/main/webapp/images/upload_process.jsp b/sns/src/main/webapp/images/upload_process.jsp
deleted file mode 100644
index d5d7976..0000000
--- a/sns/src/main/webapp/images/upload_process.jsp
+++ /dev/null
@@ -1,42 +0,0 @@
-<%@ page language="java" contentType="text/html; charset=UTF-8"
-    pageEncoding="UTF-8"%>
-<%@ page import="com.oreilly.servlet.MultipartRequest" %>
-<%@ page import="com.oreilly.servlet.multipart.DefaultFileRenamePolicy" %>
-<!DOCTYPE html>
-<html>
-<head>
-<meta charset="UTF-8">
-<title>업로드 처리</title>
-</head>
-<body>
-<%
-	request.setCharacterEncoding("UTF-8");
-	String savePath = "/mvcspring/images";
-	int uploadFileSizeLimit = 5*1024*1024;
-	String encType = "UTF-8";
-	
-	ServletContext context = getServletContext();
-	String uploadFilePath = context.getRealPath(savePath);
-	try{
-		MultipartRequest multi = new MultipartRequest(
-				request,
-				uploadFilePath,
-				uploadFileSizeLimit,
-				encType,
-				new DefaultFileRenamePolicy());
-		String fileName = multi.getFilesystemName("uploadFile");
-		if(fileName ==null){
-			System.out.print("업로드 실패");
-		}else{
-			
-		}
-	}catch(Exception e){
-		e.printStackTrace();
-	}
-
-
-%>
-
-
-</body>
-</html>
\ No newline at end of file
